pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "follow-the-gap"
+dynamic = ["version"]
```

Comment:
Look into how to set the version dynamically with `hatch-vcs`. Here's a sample [`pyproject.toml`](https://github.com/kpfleming/jinjanator/blob/7db2b4aab5d0af6b8375523d5f41d0eb62052014/pyproject.toml) that does this. You'll need to convert your project into an installable module (i.e. move your `main.py` under `src/follow-the-gap/__main__.py`). Ask @pyang4071 about this if you're unsure about how to get it to work.